alchemist-loading/src/main/kotlin/it/unibo/alchemist/boundary/extractors/NetworkCentroid.kt

```diff
@@ -0,0 +1,35 @@
+package it.unibo.alchemist.boundary.extractors
+
+import it.unibo.alchemist.boundary.Extractor
+import it.unibo.alchemist.model.Actionable
+import it.unibo.alchemist.model.Environment
+import it.unibo.alchemist.model.Time
+
+/**
+ * Exporter for the network centroid coordinates.
+ * The x and y coordinates are the average of the x and y coordinates of the nodes in the network,
+ * note that negative coordinates are possible.
+ */
+class NetworkCentroid : Extractor<Double> {
+    private companion object {
+        private const val NAME: String = "network-centroid"
+    }
+
+    override val columnNames: List<String>
+        get() = listOf<String>("$NAME-xCoord", "$NAME-yCoord")
```

Comment:
move to the companion object. remove get(), and also it is listOf("x", "y").map { "$NAME-$it" } 

what happens with 3d spaces?